#load libraries I am using
library(msa)
library(Biostrings)
library(seqinr)
library(phangorn)
library(tidyr)
library(dplyr)
# go to working directory with sequence of homo-sapiens
setwd("~/Documents/GitHub/Bioinformatics/sequences-2.fasta")
# attach files and assign each a variable
# perform Multiple Sequence Alignment
seqmid <- readDNAStringSet("sequences-2.fasta")
seqmid
myAlignment <- msa(seqmid, "ClustalW")
myAlignment
# Gene is Hemoglobin Subunit Beta
# Translate sequence to protein 
Midseq1 <- ("AATCTACTCCCAGGAGCAGGGAGGGCAGGAGCCAG 
            GCTGGGCATGAAAGTCAGGGCAGAGCCATCTATTGCTTACATTTGCTTCTGACACAACTGTGTTCACTAG
            CAACCTCAAACAGACACCATGGTGCACCTGACTCCTGTGGAGAAGTCTGCCGTTACTGCCCTGTGGGGCA
            AGGTGAACGTGGATGAAGTTGGTGGTGAGGCCCTGGGCAGGTTGGTATCAAGGTTACAAGACAGGTTTAA
            GGAGACCAATAGAAACTGGGCATGTGGAGACAGAGAAGACTCTTGGGTTTCTGATAGGCACTGACTCTCT
            CTGCCTATTGGTCTATTTTCCCACCCTTAGGCTGCTGGTGGTCTACCCTTGGACCCAGAGGTTCTTTGAG
            TCCTTTGGGGATCTGTCCACTCCTGATGCTGTTATGGGCAACCCTAAGGTGAAGGCTCATGGCAAGAAAG
            TGCTCGGTGCCTTTAGTGATGGCCTGGCTCACCTGGACAACCTCAAGGGCACCTTTGCCACACTGAGTGA
            GCTGCACTGTGACAAGCTGCACGTGGATCCTGAGAACTTCAGGGTGAGTCTATGGGACCCTTGATGTTTT
            CTTTCCCCTTCTTTTCTATGGTTAAGTTCATGTCATAGGAAGGGG")
Midseq1 <- DNAString("
AATCTACTCCCAGGAGCAGGGAGGGCAGGAGCCAGG
GCTGGGCATGAAAGTCAGGGCAGAGCCATCTATTGCTTACATTTGCTTCTGACACAACTGTGTTCACTAG
CAACCTCAAACAGACACCATGGTGCACCTGACTCCTGTGGAGAAGTCTGCCGTTACTGCCCTGTGGGGCA
AGGTGAACGTGGATGAAGTTGGTGGTGAGGCCCTGGGCAGGTTGGTATCAAGGTTACAAGACAGGTTTAA
GGAGACCAATAGAAACTGGGCATGTGGAGACAGAGAAGACTCTTGGGTTTCTGATAGGCACTGACTCTCT
CTGCCTATTGGTCTATTTTCCCACCCTTAGGCTGCTGGTGGTCTACCCTTGGACCCAGAGGTTCTTTGAG
TCCTTTGGGGATCTGTCCACTCCTGATGCTGTTATGGGCAACCCTAAGGTGAAGGCTCATGGCAAGAAAG
TGCTCGGTGCCTTTAGTGATGGCCTGGCTCACCTGGACAACCTCAAGGGCACCTTTGCCACACTGAGTGA
GCTGCACTGTGACAAGCTGCACGTGGATCCTGAGAACTTCAGGGTGAGTCTATGGGACCCTTGATGTTTT
CTTTCCCCTTCTTTTCTATGGTTAAGTTCATGTCATAGGAAGGGG")
Midseq1
library(Biostrings)
# fasta file
Midseq1 <- Biostrings::readDNAStringSet("file.fasta", format = "fasta", use.names = TRUE)
seqmid <- Biostrings::readDNAStringSet("file.fasta", format = "fasta", use.names = TRUE)
#3d picture attached to document. 
# Use Uniprot to figure out the protein
# Protein: Hemoglobin Subunit Beta
# Diseases associated: Sickle Cell, Anemia, Malaria Resistance etc
# 3D picture added to doc. 